### ISSUE TITLE
dsserve not found when building latest master for broadcom  

### ISSUE THREAD

**Issue by ShaabamRouter** [sonic-net/sonic-buildimage#20695]
#### Description
I downloaded the latest master last night but when I go to build it fails while trying to setup gbsyncd-broncos.

#### Steps to reproduce the issue:
1. git clone --recurse-submodules https://github.com/sonic-net/sonic-buildimage.git
2. cd sonic-buildimage/
3. sudo modprobe overlay
4. make init
5. make configure PLATFORM=broadcom
6. make SONIC_BUILD_JOBS=4 all
7. build fails saying it can't find dsserve that's its supposed to copy to /usr/bin

If I look in platform/components/docker-gbsyncd-broncos I see the other files like bcmsh just not dsserve.
```
/SONiC/sonic-buildimage/platform/components/docker-gbsyncd-broncos$ ls -al
total 64
drwxrwxr-x 8 lab lab 4096 Nov  4 16:43 .
drwxrwxr-x 4 lab lab 4096 Nov  4 00:02 ..
-rwxrwxr-x 1 lab lab  862 Nov  4 00:02 bcmsh
drwxr-xr-x 4 lab lab 4096 Nov  4 16:43 buildinfo
-rw-rw-r-- 1 lab lab   30 Nov  4 00:02 critical_processes.j2
drwxr-xr-x 2 lab lab 4096 Nov  4 15:42 debs
-rw-r--r-- 1 lab lab 1975 Nov  4 16:43 Dockerfile
-rw-rw-r-- 1 lab lab 1039 Nov  4 00:02 Dockerfile.j2
-rwxrwxr-x 1 lab lab  435 Nov  4 00:02 docker-init.sh
drwxr-xr-x 2 lab lab 4096 Nov  4 15:42 files
-rw-r--r-- 1 lab lab  769 Nov  4 16:43 manifest.json
drwxr-xr-x 2 lab lab 4096 Nov  4 15:42 python-debs
drwxr-xr-x 2 lab lab 4096 Nov  4 15:42 python-wheels
-rwxrwxr-x 1 lab lab  372 Nov  4 00:02 start.sh
-rw-rw-r-- 1 lab lab 1001 Nov  4 00:02 supervisord.conf.j2
drwxrwxrwx 2 lab lab 4096 Nov  4 15:42 vcache
```

build machine is ubuntu 22.04
Docker version 27.0.3, build 7d4bcd8

Any idea what I might be doing wrong?

#### Describe the results you received:
```
Unpacking libsaibroncos (3.11) ...
Setting up libnl-3-200:amd64 (3.5.0-1) ...
Setting up libnl-route-3-200:amd64 (3.5.0-1) ...
Setting up libnl-genl-3-200:amd64 (3.5.0-1) ...
Setting up libnl-nf-3-200:amd64 (3.5.0-1) ...
Setting up libnl-cli-3-200:amd64 (3.5.0-1) ...
Setting up libyang (1.0.73) ...
Setting up libswsscommon (1.0.0) ...
Setting up libsairedis (1.0.0) ...
Setting up libsaimetadata (1.0.0) ...
Setting up syncd (1.0.0) ...

Configuration file '/etc/init.d/syncd', does not exist on system.
Installing new config file as you requested.
invoke-rc.d: could not determine current runlevel
invoke-rc.d: policy-rc.d denied execution of start.
Setting up libsaibroncos (3.11) ...
Processing triggers for libc-bin (2.31-13+deb11u11) ...
Removing intermediate container 87b66d8725be
 ---> c03078dc278b
Step 16/32 : COPY ["docker-init.sh", "/usr/bin/"]
 ---> e74a4e738018
Step 17/32 : COPY ["start.sh", "/usr/bin/"]
 ---> ed3043b117e7
Step 18/32 : COPY ["bcmsh", "/usr/bin/"]
 ---> d1749a2a03fe
Step 19/32 : COPY ["critical_processes.j2", "/usr/share/sonic/templates"]
 ---> 31d163629fc2
Step 20/32 : COPY ["supervisord.conf.j2", "/usr/share/sonic/templates"]
 ---> 77b298d37c41
Step 21/32 : COPY ["files/supervisor-proc-exit-listener", "/usr/bin"]
 ---> 33e4a99f250a
Step 22/32 : COPY ["files/dsserve", "/usr/bin/"]
COPY failed: file not found in build context or excluded by .dockerignore: stat files/dsserve: file does not exist
[  FAIL LOG END  ] [ target/docker-gbsyncd-broncos.gz ]
make: *** [slave.mk:1130: target/docker-gbsyncd-broncos.gz] Error 1
make: *** Waiting for unfinished jobs....
make[1]: *** [Makefile.work:605: bullseye] Error 2
make[1]: Leaving directory '/home/lab/SONiC/sonic-buildimage'
make: *** [Makefile:49: all] Error 2
```

**Comment by gord1306** [comment]
This issue might be related to https://github.com/sonic-net/sonic-buildimage/issues/20685

**Comment by Pan-XT** [comment]
https://github.com/sonic-net/sonic-buildimage/pull/20696

**Comment by wdoekes** [comment]
Yea, the merged PR #20696 is the fix here, AFAICT. Close?

**Comment by prgeor** [comment]
@ShaabamRouter please check the latest master and try
